src/Workspaces/Core/MSBuild/MSBuild/Build/ProjectBuildManager.cs

```diff
@@ -31,13 +31,7 @@ internal class ProjectBuildManager
             { PropertyNames.DesignTimeBuild, bool.TrueString },
 
             // this will force CoreCompile task to execute even if all inputs and outputs are up to date
-#if NETCOREAPP
```

Comment:
We were hitting https://github.com/dotnet/msbuild/issues/4770, presumably because https://github.com/dotnet/msbuild/pull/5045 wasn't available in one or more SDKs we were executing against.